### DISCUSSION TITLE
Is is possible to convert a *.bim file into *.dgn?

### DISCUSSION THREAD

**Discussion by Snailya** [iTwin/itwinjs-core#3568] [Q&A]
iTwin Snapshot and MicroStation could convert a dgn file to bim file which could be used as snapshot. 
But after some operation in iTwin, how could I convert it back to dgn that benifits our design.

**Comment by kabentley** [comment]
attach the iModel to MicroStation and do a "save as".

**Reply by Snailya** [reply]
![image](https://user-images.githubusercontent.com/19658624/167052339-efdf9889-a7e1-4b29-8dcc-d82258efce0a.png)
It seems that attach a bim file as reference is not allowed.

**Reply by ColinKerr** [reply]
iTwin Attach was added to MicroStation CONNECT Edition Update 15.  Do you have Update 15 or greater?

Here is a link to CONNECT edition docs for iTwin Attach: https://docs.bentley.com/livecontent/web/microstation%20help-v20/en/GUID-08F38DB6-9A69-42B6-8ACD-6B06176CD3E1.html
